.goreleaser.yml

```diff
@@ -31,10 +31,72 @@ archives:
     format_overrides:
       - goos: windows
         format: zip
+snapcrafts:
+  - name: parca
+    # Switch to true once snap ownership is sorted
+    publish: false
+    summary: Parca continuous profiling tool
+    description: |
+      Continuous profiling for analysis of CPU and memory usage, down to the line
+      number and throughout time. Saving infrastructure cost, improving performance,
+      and increasing reliability.
+    # This allows control over when the snap is published into `stable`
+    grade: "{{ if .IsSnapshot }}devel{{ else }}stable{{ end }}"
```

Comment:
I've gone for this approach at the moment, because I can't see an obvious way to control the releases such that snapshots don't end up in `stable`. Got an open question with the goreleaser folk so will revisit once I get an answer.

Reply:
This didn't age well :sweat_smile: